optaplanner-workbench-models/optaplanner-workbench-models-core/pom.xml

```
@@ -7,7 +7,7 @@
   <parent>
     <groupId>org.optaplanner</groupId>
     <artifactId>optaplanner-workbench-models</artifactId>
-    <version>7.75.0-SNAPSHOT</version>
+    <version>${version.org.optaplanner}</version>
```

Here we have to keep the current snapshot version because it refers to the parent pom built here. Furtheremore we cannot use properties inside parent poms.
```suggestion
    <version>7.75.0-SNAPSHOT</version>
```